### PR TITLE
2 02 validation

### PR DESCRIPTION
**Pull request by amilan17**
Please review this branch before I merge it with the FT21-1-validation branch. 

Go to 'Files changed' tab just above this box and enter your review there. 

I believe that I captured all of the decisions in the 10 issues for amending table 2-02.csv, but there were conflicts that I had to adjudicate manually, which could have introduced errors. 

https://github.com/wmo-im/wmds/issues?q=is%3Aissue+is%3Aopen+label%3A2-02-validation

### CODE REVIEW

**Review by fstuerzl — COMMENTED** [review]
@amilan17 looks good!

**Review by ferrighi — COMMENTED** [review]
All changes look good to me, and reflect the open issues on 2-02 table as far as I can see. I only have one comment about the consistency of the naming for GDP-ESURFMAR. As similar entries have the structure E-SURFMAR-X instead. Can you double check this?

@ferrighi  - I added a dash to be consistent with other E-SURFMAR... values.

File: tables_en/2-02.csv [hunk 1]
```diff
@@ -1,22 +1,20 @@
 notation,path,name,description
-ABOS,\WIGOS\GOS\GOS Other elements\ABOS,ABOS,Aircraft-Based Observing Systems
 ACTRIS,\WIGOS\GAW\GAW Other elements\ACTRIS,ACTRIS,"Aerosol,Clouds and Trace Gases Research Infrastructure, www.actris.eu"
 ADD,\WIGOS\GOS\GOS Other elements\ABOS\ADD,ADD,Aircraft Derived Data (Mode-S/ADS-B)
-ADNet,\WIGOS\GAW\GAW Contributing networks\GALION\ADNet,ADNet,AD-Net Asian Dust and Lidar Observation Network
+ADNet,\WIGOS\GAW\GAW Contributing networks\ADNet,ADNet,AD-Net Asian Dust and Lidar Observation Network
 ADSC,\WIGOS\GOS\GOS Other elements\ABOS\ADS-C,ADS-C,Automatic Data Surveillance (ICAO). These are transmitted at selected points of the flightpath.
-AEROCAN,\WIGOS\GAW\GAW Other elements\AOD\AEROCAN,AEROCAN,"AEROCAN, federated to AERONET"
-AERONET,\WIGOS\GAW\GAW Other elements\AOD\AERONET,AERONET,AErosol RObotic NETwork
+AEROCAN,\WIGOS\GAW\GAW Other elements\AEROCAN,AEROCAN,"AEROCAN, federated to AERONET"
```
Comment (joergklausen) [inline tables_en/2-02.csv:6]: @amilan17  Not sure if double-quotes should be removed or inserted everywhere else.

Comment (amilan17) [reply tables_en/2-02.csv:6]: The double quotes are in this line to ensure the integrity of the comma-delimited CSV. They are required when there is a comma within the text of a cell in the table. It is not necessary when there are no commas.

File: tables_en/2-02.csv [hunk 2]
```diff
@@ -332,31 +317,26 @@ OSWHOI,\Co-sponsored\GOOS\OceanSITES\OS-WHOI,OS-WHOI,NA
...
 PMELMB,\Co-sponsored\GOOS\DBCP\PMEL MB,PMEL MB,PMEL Moored Buoys
-polarAOD,\WIGOS\GAW\GAW Other elements\AOD\PolarAOD,PolarAOD,Polar Aerosol Optical Depth Measurement Network Project
-RBCN,\WIGOS\GOS\GOS Surface networks\RBCN,RBCN,Regional Basic Climatological Networks
-RBONpilot,\WIGOS\GOS\GOS Surface networks\RBON-Pilot Project,RBON-Pilot Project,Regional Basic Observing Network (pilot project)
-RBSN,\WIGOS\GOS\GOS Surface networks\RBSN,RBSN,GOS Regional Basic Synoptic Networks
-RBSNp,\WIGOS\GOS\GOS Surface networks\RBSN\RBSN(P),RBSN(P),Regional Basic Synoptic Network upper-air station (PILOT)
-RBSNs,\WIGOS\GOS\GOS Surface networks\RBSN\RBSN(S),RBSN(S),Regional Basic Synoptic Network surface station (SYNOP)
-RBSNsp,\WIGOS\GOS\GOS Surface networks\RBSN\RBSN(SP),RBSN(SP),NA
-RBSNst,\WIGOS\GOS\GOS Surface networks\RBSN\RBSN(ST),RBSN(ST),Regional Basic Synoptic Network surface and upper-air station (SYNOP/TEMP)
-RBSNt,\WIGOS\GOS\GOS Surface networks\RBSN\RBSN(T),RBSN(T),Regional Basic Synoptic Network upper-air station (TEMP)
```
Comment (joergklausen) [inline tables_en/2-02.csv:345]: @amilan17 So, all these RBCN and RBSN entries will be listed as superseded by RBON, correct? Are we supposed to re-assign all existing affiliations in OSCAR/Surface to RBON as well, or will we keep them? No immediate response required, but we need to agree how OSCAR/Surface should foow-up such changes.

**Review by joergklausen — APPROVED** [review]

**Comment by Netcheva** [comment]
Oksana‘s direction for AERONET as of yesterday was AERONET to not be linked to GAW – can go under WIGOS or wherever AOD is measured as main variable.

**Comment by joergklausen** [comment]
@Netcheva So I propose we list AERONET under
/Non-Affiliated/AERONET
at the top of the hierarchy (where it is presently in OSCAR/Surface.
What about AEROCAN, then? Same thing?

@amilan17 @fstuerzl Please proceed like that if @Netcheva confirms. This has been going back and forth too many times, apologies.

**Comment by Netcheva** [comment]
During the discussion between 4 of us we agreed to keep AEROCAN under GAW Other elements.

**Comment by fstuerzl** [comment]
@amilan17, I lost track of all the different branches... is this the correct branch, where the change @joergklausen and @Netcheva requested should be made: https://github.com/wmo-im/wmds/blob/2-02-validation/tables_en/2-02.csv?

File: tables_en/2-02.csv (hunk 2 above)
Comment (amilan17) [reply tables_en/2-02.csv:345]: @joergklausen  - Yes, these terms are superseded by RBON and this will be apparent in the Codes Registry.

Comment (amilan17) [reply tables_en/2-02.csv:345]: @joergklausen - please resolve this conversation if you are satisfied with my answer.

File: tables_en/2-02.csv (hunk 1 above)
Comment (amilan17) [reply tables_en/2-02.csv:6]: @joergklausen - please resolve this conversation if you are satisfied with my response.

**Comment by amilan17** [comment]
I will make all changes moving forward. thanks.

Anna Milan
Scientific Officer
Data and Information Management Division
Infrastructure Department
World Meteorological Organization

________________________________

**Comment by amilan17** [comment]
AEROCAN,\WIGOS\GAW\GAW Other elements\AEROCAN,AEROCAN,"AEROCAN, federated to AERONET"
AERONET,\WIGOS\GAW\GAW Other elements\AERONET,AERONET,AErosol RObotic NETwork

@Netcheva @joergklausen I need confirmation that these terms should remain as is. 

As stated in @Netcheva's last comment: 
> During the discussion between 4 of us we agreed to keep AEROCAN under GAW Other elements.

**Comment by Netcheva** [comment]
I confirm: AEROCAN,\WIGOS\GAW\GAW Other elements\AEROCAN,AEROCAN,

For AERONET:  AERONET,\Non-affiliated\AERONET,AERONET,AErosol RObotic NETwork
or whatever the proper synaxis is for the 2d one. Basically leave it as it shows now in OSCAR - as in attached screen shot.   
AEROCAN in OSCAR

**Review by amilan17 — COMMENTED** [review]
all recommended changes are included in table 2-02

**Pull request merged**